todo_app/data/session_items.py

```diff
@@ -1,20 +1,35 @@
-from flask import session
+from flask import Flask
+from requests import get, post, put
+from todo_app.flask_config import Config
+from todo_app.data.ToDoItem import ToDoItem 
 
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
+app = Flask(__name__)
+app.config.from_object(Config)
```

Comment:
Why is this also trying to be a flask app?

Reply:
Removed and referred back to Config object in commit 4f9b126